src/DataFrames.jl

```diff
@@ -80,6 +80,13 @@ if VERSION < v"1.2"
     export hasproperty
 end
 
+if isdefined(Base, :only)  # Introduced in 1.4.0
```

Comment:
I don't to prefer explicit VERSION checks, which Femtocleaner can automatically remove once we drop support for 1.4, but as you like.